### PR TITLE
COMMERCE-11927

### PR DESCRIPTION
**Pull request by victorcdp**
https://liferay.atlassian.net/browse/COMMERCE-11927

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:Sf

**Comment by victorcdp** [comment]
ci:test:sf

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java [hunk 1]
```diff
@@ -191,6 +198,41 @@ public List<AccountGroupRel> getAccountGroupRelsByAccountGroupId(
 			accountGroupId, start, end, orderByComparator);
 	}
 
+	@Override
+	public List<AccountGroupRel> getAccountGroupRelsByAccountGroupName(
+		String keywords, long cpDefinitionId, int start, int end) {
```
Comment (brianikim) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java]: You need to implement this at the `service` layer not just, `localService`. Also, are there any examples of code that uses similar naming scheme as this?

Comment (victorcdp) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java]: @brianikim there's another method above this one, getAccountGroupRelsByAccountGroupId and for the Channel filter, the method is just getCommerceChannelRels, so i don't know which one to follow.

Comment (brianikim) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java]: Look for examples in code please, including non-commerce code.

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java [hunk 2]
```diff
@@ -157,6 +164,41 @@ public List<AccountGroupRel> getAccountGroupRels(
 			accountGroupId, _classNameLocalService.getClassNameId(className));
 	}
 
+	@Override
+	public List<AccountGroupRel> getAccountGroupRels(
+		long cpDefinitionId, String keywords, int start, int end) {
```
Comment (brianikim) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java]: This is wrong already. Do not use commerce related naming schemes in non-COMMERCE modules. It needs to be `long classPK`.

Comment (brianikim) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java]: Also, you should also match `className` in the query. That was being provided as a parameter here: https://github.com/brianikim/liferay-portal/pull/98/commits/97b40c888f79dfbbb4eefac1ec8bbc039b980113 which you should not remove.

Comment (brianikim) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java]: Expected is:
```
String className, long classPK, String keywords, int start, int end) {
```

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java [hunk 3]
```diff
@@ -175,6 +182,47 @@ public List<AccountGroupRel> getAccountGroupRels(
...
+					AccountGroupRelTable.INSTANCE.accountGroupId)
+			).where(
+				() -> {
+					Predicate predicate =
+						AccountGroupRelTable.INSTANCE.classPK.eq(
+							classPK
+						).and(
+							AccountGroupRelTable.INSTANCE.classNameId.eq(
+								classNameId)
+						);
```
Comment (brianikim) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java:205]: Wrong order.

Comment (brianikim) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java:205]: Also, pass in the method call for `getClassNameId` directly please. no need for variable.

**Pull request closed without merging**